lms/templates/ccx/underscore/right-container.underscore

```diff
@@ -0,0 +1,30 @@
+<% if(unsavedChanges) { %>
+   <div class="ccx-sidebar-panel" id="dirty-schedule" tabindex="-1" role="region"
+           aria-labelledby="ccx_schedule_save_changes_heading">
+     <h2 id="ccx_schedule_save_changes_heading"><%- gettext('Save changes') %></h2>
+     <form>
+       <p id="message_save" class="text-helper"><%- gettext('You have unsaved changes.') %></p>
+       <div class="field">
+         <br/>
+         <button id="save-changes"
+           aria-describedby="message_save"
+           class="ccx-schedule-save-changes">
+           <%- gettext('Save changes') %>
+         </button>
+       </div>
+     </form>
+   </div>
+<% } %>
+
+<div class="ccx-sidebar-panel" aria-labelledby="ccx_schedule_unit" id="ccx_schedule_unit_panel" role="region">
+ <h2 id="ccx_schedule_unit"><%- gettext('Schedule a Unit') %></h2>
+ <% if (chaptersAvailable) { %>
+   <form role="form" id="add-unit" name="add-unit" class="ccx-form">
+     <%= form %>
```

Comment:
Because this form is actual HTML, the usage of <%= is correct. However, there are 2 best practices--

1) Name "form" something that is obviously HTML (formHtml, for example).
2) Wrap the variable with HtmlUtils.ensureHTML. This requires usage of HtmlUtils where the variable was defined.